### ISSUE TITLE
Arithmetic Overflow Error During Profiler Ingestion

### ISSUE THREAD

**Issue by rexdev88** [open-metadata/OpenMetadata#14044]
**Affected module**
Profiler Ingestion

**Describe the bug**
The issue occurs when doing profiler ingestion. Particularly when ingesting fields that has data type datetime.  

**To Reproduce**
![image](https://github.com/open-metadata/OpenMetadata/assets/151584836/0c1431d3-a54e-46fd-9038-3ee9dbc5e4d0)

**Version:**
- OpenMetadata 1.1.6
- Source DB: SQL Server 2019
- OpenMetadata Ingestion package version: openmetadata-ingestion[mysql]~=1.1.6

**Comment by harshach** [comment]
@TeddyCr can you take a look

**Comment by TeddyCr** [comment]
@rexdev88 could you try a few things please?
1. update to the latest stable version and run the profiler again -- let us know if the error is still present

if the error is still present after doing 1. could you then:
1. past the error log
2. share MSSQL version
3. share the schema with us

I tried to reproduce it with the below schema and version and I am not seeing any error


```
Microsoft Azure SQL Edge (RTM) - 15.0.2000.1574 (X64) 
	Jan 25 2023 11:33:34 
	Copyright (C) 2019 Microsoft Corporation
	Linux (Ubuntu 18.04.6 LTS) <X64>
```



![Image](https://github.com/open-metadata/OpenMetadata/assets/13626425/eaf839a5-ca69-43d3-9a6c-b2021e9623c9)

**Comment by rexdev88** [comment]
@TeddyCr this issue has been resolved in 1.2.3.
Thank you for looking into this.

**Comment by rex-devilla** [comment]
Did a profiler ingestion and this issue reoccurred. Previously, the test was done in stage and issue seems to be resolved, then we moved to prod and had the issue again:
OM: 1.2.3
SQL: 2019

![image](https://github.com/open-metadata/OpenMetadata/assets/92232857/4c98cd0d-bc7f-49c5-91af-560f9d1e1133)
![image](https://github.com/open-metadata/OpenMetadata/assets/92232857/76e567cb-d1bc-4b89-ace3-a3a2e1584f75)
